userguide/tutorials/plugin_introduction.adoc

```diff
@@ -62,6 +62,137 @@ Kill Bill Plugins have full power (and therefore need to be designed and tested
 * They receive all events in the system.
 * They are isolated from the rest of the code and can use their own libraries (versions) without risk of conflict.
 
+== List of Open-Source Plugins
```

Comment:
Should we merge this with the `Examples of Plugins` section above?

Reply:
@pierre - Yes, I needed to do more work on this, didn't I? ;-)